lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -127,7 +127,8 @@ class DetailsRenderer {
       displayedHost = parsed.file === '/' ? '' : `(${parsed.hostname})`;
       title = url;
     } catch (/** @type {!Error} */ e) {
-      if (!e.name.startsWith('TypeError')) {
+      // All browsers throw a TypeError except Edge they throw a regular Error with Invalid argument as a message
+      if (!e.name.startsWith('TypeError') && !e.message.startsWith('Invalid argument')) {
```

Comment:
i think we could afford to remove this `if (...) { throw e; }` block entirely.

Here's the history:

1. https://github.com/GoogleChrome/lighthouse/pull/2312#discussion_r118729232
1. https://github.com/GoogleChrome/lighthouse/pull/5386#discussion_r192170145

From what I can tell, we might as well just discard any exception here. Throwing at this point seems.. bad. :)

@patrickhulce sgty?